### DISCUSSION TITLE
1.25.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#12011] [General]
# ~~Today marks the beginning of a new era~~
~~We are very excited to share this news with you, especially since we have been working on this for a long time. You probably already heard about the upcoming [Matter protocol](https://csa-iot.org/all-solutions/matter/) which aims to provide a unified wireless protocol. We decided to hop on this train, meaning that starting from today **Zigbee**2MQTT will become **Matter**2MQTT!~~

<img src="https://user-images.githubusercontent.com/2892853/160241082-70afcd13-836b-43d4-bad8-fc86d3a7d416.png" width="500">

~~What does this mean for me?~~
- ~~The migration is automatic, when upgrading to this version the Zigbee firmware on your adapter will be replaced with a Matter firmware~~
- ~~All your Zigbee devices will stop working (we hope this is not a problem for you)~~
- ~~All Matter devices in your home will be paired automatically~~
- ~~Since the Matter protocol has been [delayed for the third time](https://9to5mac.com/2022/03/17/matter-has-been-delayed-yet-again/#:~:text=Matter%20delayed%20for%20the%20third%20time&text=Expected%20this%20summer%2C%20the%20launch,products%20into%20the%20Matter%20ecosystem.), expect low stability, we are still learning how to deal with Matter :smile:~~

**Update: happy Aprils fools day!**

# Notes
- **Important** for users running **Raspbian Buster**: due to #11804, `libseccomp2` is required which needs to be installed manually. If you do not do this the container will crash on startup. See [the docs](https://www.zigbee2mqtt.io/guide/installation/02_docker.html#notes-for-raspberry-pi-users) on how to do this.
- A new firmware for adapters based on the CC1352 and CC2652 has been released (`20220219`) ([changelog](https://github.com/Koenkk/Z-Stack-firmware/releases/tag/Z-Stack_Home_3.x.0_coordinator_20220219)). Go to the [Supported adapters page](https://www.zigbee2mqtt.io/guide/adapters), your adapter and click on "Coordinator firmware" to retrieve it.

# Features
- #9176 Availability for groups ([docs](https://www.zigbee2mqtt.io/guide/configuration/device-availability.html#groups))

# Improvements
- [#3945](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3945) Add power monitoring support to Enbrighten 43082
- [#3946](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3946) [#3951](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3951) More functionality for ZLinky_TIC
- [#3938](https://github.com/Koenkk/zigbee-herdsman-converters/issues/3938) Expose lock actions for all locks supporting it
- [#3958](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3958) Support power measurements for Sengled E21-N1EA and E1F-N5E and enable OTA for more Sengled devices
- [#3961](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3961) Expose system_mode emergency_heating for Hive SLR2/b/c
- [#3963](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3963) Support additional gangs and new firmware for Livolo TI0001
- #11183 Don't cache `no_occupancy_since`
- #9826 Expose battery and voltage for Woox R7060
- [#3968](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3968) Expose battery & position for Qmotion blinds HDM40PV620
- [#3967](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3967) Allow to control all 3 switches of Lidl HG06338
- [#3966](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3966) Expose `motion_sensitivity` and `reset_nopresence_status` for Xiaomi RTCZCGQ11LM
- #11381 Expose backlight LED brightness for Aurora Lighting AU-A1ZBDSS
- #11804 Upgrade Docker container to Node.js 16 and Alpine 3.15
- [#3972](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3972) Expose `power_outage_count` for Xiaomi RTCZCGQ11LM, JT-BZ-01AQ/A, LLKZMK11LM and SSM-U01
- [#3980](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3980) Expose `running` for TuYa TS0601_cover, Xiaomi ZNCLDJ11LM, SRSC-M01 and ZNCLDJ12LM
- #11789 #11926 Expose cover running/motor state to Home Assistant (allows HA to stop cover)
- [#3989](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3989) Expose battery and battery_voltage for Third Reality 3RSS008Z
- [#3993](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3993) Support `power_outage_memory` for Xiaomi LLKZMK11LM
- [#3992](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3992) Expose `window_open` for TuYa TS0601_smart_air_house_keeper and TS0601_thermostat
- [#4008](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4008) Expose `battery` and `charging_status` for Xiaomi ZNJLBL01LM
- [#4006](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4006) Add `occupancy_timeout` and `no_occupancy_since` options, as well `temperature` expose for Xiaomi RTCGQ12LM and RTCGQ13LM
- [#4026](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4026) Expose `power_outage_memory` for Xiaomi SP-EUC01
- #11119 Support battery and fix configure failing for TuYa TS0601_smoke
- [#3887](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3887) Expose `battery_voltage` for TuYa TS0203
- [#4029](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4029) Expose energy and fix configure and reporting for Niko 170-33505
- [#4027](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4027) Improve `running` for Xiaomi Covers and add `motor_state` for ZNCLDJ11LM
- [#4020](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4020) Expose `preheat_status`, `adaptation_run_status`, `adaptation_run_settings`, `adaptation_run_control` and `regulation_setpoint_offset` for Danfoss 014G2461
- [#4036](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4036) Expose lock, unlock, half_open and tilt for EnOcean PTM 215ZE
- [#4047](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4047) Support min and max brightness for TuYa TS0601_dimmer

# Fixes
- #11675 Fix TS0201 battery (`fz.battery is not a function` error)
- #11645 Fix colour symbols appearing in non coloured log
- #11217 Remove color support for Philips 929003055901 since device doesn't support it
- [#3952](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3952) Fix left/right inversed for Legrand 067772
- [#3820](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3820) Remove battery and battery_low expose for mains powered TuYa TS0204
- #11470 Fix TuYa TS0201 battery reporting
- #11706 Fix power measurements reporting not working for some SmartThings STS-OUT-US-2
- [#3956](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3956) Fix power measurements not reported for Xiaomi LLKZMK11LM with newer firmwares
- #11777 Limit Saswell SEA801-Zigbee/SEA802-Zigbee local temperature calibration range from -6 to 6
- [#3965](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3965) Fix AwoX 33952 not being identified
- [#3976](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3976) Fix Orvibo AM25 power source
- [#3978](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3978) Fix local water temperature not reported for Hive SLR2, SLR2b and SLR2c
- [#3982](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3982) Fix co2, ch2o and pm25 incorrect for TuYa TS0601_smart_air_house_keeper
- [#4002](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4002) Fix Schneider Electric EKO09738 power measurements
- #11864 Fix device reconnect resulting in brightness 1 when device is off
- #10973 Fix configure failing when rejoining device
- #11872 Fix UK7004240 fails to configure
- #11392 Fix nothing reported for TuYa TS0222
- [#4037](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4037) Fix inverted Xiaomi covers controls
- [#4039](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4039) Fix Xiaomi cover position incorrect when not fully opened or closed
- [#4040](https://github.com/Koenkk/zigbee-herdsman-converters/issues/4040) Fix TuYa cover `_TZE200_rmymn92d` position incorrect (inverted)
- #11915 Enable H/S colour for 33943/33944/33946 and fix colour temperature range
- #11663 Fix battery reporting not working for TuYa TS0202
- #9400 Fix TuYa TS0201 not reporting battery percentage

# New supported devices
This release adds support for **43** new devices:
- `046677562229` Philips Hue White PAR20 with Bluetooth
- `1001923` SLV VALETO LED GU10 RGBW
- `106-03` Plugwise Tom thermostatic radiator valve
- `14158704L` Lidl Livarno Home LED floor lamp, RGBW
- `1746530P7` Philips Hue White and color ambiance Daylo outdoor wall lamp
- `3025CSGZ` Schneider Electric Dual connected smart socket
- `4023331P6` Philips Hue white ambiance Amaze
- `4512737` Namron Touch termostat
- `45391` Müller Licht Smart power strip
- `511.544` Iluminize Zigbee 3.0 wall dimmer RGBW 4 zone
- `552-721X1` Niko Single connectable switch
- `552-721X2` Niko Double connectable switch
- `801480` Philips Hue white ambiance 4" retrofit recessed downlight
- `81809FBA` AduroSmart ERIA colors and white shades smart light bulb A19/BR30
- `90504044` IKEA NYMÅNE Pendant lamp
- `915005997001` Philips Hue white ambiance ceiling light Enrave XL with Bluetooth
- `929002335105` Philips Hue white ambiance E26 1600lm with Bluetooth
- `929002980901` Philips Hue white ambiance G40 E26 filament globe with Bluetooth
- `929003047101` Philips Hue White ambiance Milliskin (round)
- `929003055201` Philips Hue Being
- `AC33903` LEDVANCE SMART+ classic P 40 E14 tunable white
- `AC33905` LEDVANCE SMART+ spot GU10 tunable white
- `AC33906` LEDVANCE SMART+ spot GU10 multicolor RGBW
- `BY 266` Innr B22 (Bayonet) bulb, dimmable
- `CCT595011` Schneider Electric Wiser motion sensor
- `EKO05806` ELKO Elko ESH 316 Endevender RF
- `HG08164` Lidl Silvercrest smart button
- `IG-CDZFB2AG010RA-MNZ` ADEO ENKI LEXMAN E27 LED white
- `IM-CDZDGAAA0005KA8MAN` ADEO ENKI LEXMAN RGBTW GU10 Bulb
- `OR-ZB-S010-3C` ORVIBO Smart Socket
- `QS-Zigbee-C03` Lonsonho Curtain/blind motor controller
- `R7051` Woox Smart siren
- `RB 249 T` Innr E14 candle, dimmable with, color temp
- `RB 266` Innr E27 bulb
- `SP-PS1-02` Spotmau Smart wall switch - 1 gang
- `SP-PS2-02` Spotmau Smart wall switch - 2 gang
- `TS0002_switch_module_2` TuYa 2 gang switch module
- `TS011F_2_gang_2_usb_wall` TuYa 2 gang 2 usb wall outlet
- `WM25L-Z` Smartwings Roller shade
- `X2SK11` XingHuoYuan Smart socket
- `YMF30` Yale Digital lock
- `ZB24100VS` EVN Zigbee multicolor controller with power supply
- `ZGR904-S` Envilar Envilar touchlink remote

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.25.0'>1.25.0</a>.</em>

**Comment by andreypopov** [comment]
`All your Zigbee devices will stop working (we hope this is not a problem for you)`

So will zigbee devices work after repairing?

**Reply by lgxmedia** [reply]
It won't, forget about them. Matter is the future 😜😂

**Reply by andreypopov** [reply]
why not add a new repo for matter2mqtt ? and leave zigbee2mqtt

**Reply by andreypopov** [reply]
😂😂😂😂😂😂😂

**Comment by lgxmedia** [comment]
I was like: wtf, this is nuts, crazy, ... Piuh, good one! 😂

**Comment by mabi** [comment]
At last! Now my useless zigbee devices do not disturb WiFi anymore! 🤣

**Comment by fortax** [comment]
👍😁

**Comment by andreypopov** [comment]
AHAHHAHA!!! very funny

**Reply by ryan-sondgeroth** [reply]
Dude, you got me....you got me fair and square with this one.  Well played.

**Reply by OlafKocanda** [reply]
kind of confused - so this is an april's fool right? 😄

**Reply by pystha** [reply]
I almost passed out...
However this was suspicious _"the Zigbee firmware on your adapter will be replaced with a Matter firmware"_

**Comment by GerdRuetten** [comment]
I've do the update, and now, all old zigbee-Devices speak to me in Matter, but i don't understand them.

Is this normal?
How can i learn the Matter-Language?
Is the Matter-Language similar like esperanto?
And they speak very loudly. Is it possible to configure the volume via the Matter UI?

Questions over questions !

**Reply by robertalexa** [reply]
the answer is simple. Duolingo :)

**Comment by bfrigon** [comment]
Oh right... It's that time of the year again :)

**Comment by itavero** [comment]
Not sure if it was intentional to not publish it to NPM, but the publish action seems to have failed.

**Reply by Koenkk** [reply]
Seems something went wrong in the CI pipeline, did a manual publish now.

**Comment by sjorge** [comment]
Some people might not be too happy with the change. But it's good to look towards the future, that's all that should really Matter.

**Comment by KrzysztofHajdamowicz** [comment]
Definetely description is not funny.
It's better to be safe than sorry and don't upgrade anything on 01.04 and on Friday :D

**Comment by mijeca** [comment]
At least i know now about the existance of Matter.

**Comment by Gerrist** [comment]
@Koenkk The 90504044 IKEA light doesn't seem to work. 

I can connect it to both the hue and IKEA bridge and connect directly to it via an IKEA remote. I've tried resetting the lamp a few times (on/of 6 times) but that doesn't seem to reset the lamp. Literally stuffing the ZigBee adapter (ConBee) also doesn't work.
The manual also doesn't mention anything about resetting nor pairing

**Comment by jfmcdd** [comment]
@Koenkk 

Re first release note:
On my Pi 1B+, the libseccomp2 workaround consistently bricks the system. The install procedure seems to work fine, but once you try to reboot it no longer works. I confirmed this for myself on the most recent Buster version, just installed clean without adding anything. Anybody experiencing the same (and perhaps able to share a solution)? This prevents me from upgrading zigbee2mqtt to 1.25.x and beyond.
NB. Same approach. i.e upgrading to lipseccomb2 on a clean Buster version, does allow my Pi 3B+ to restart. Unfortunately, the thus upgraded SD does no longer work in the 1B+. Therefore this lipseccomp2 issue seems restricted to older Pis. Again if anybody can corroborate, this can be raised as an issue (as it stands I am hesitant to bother Koen with what may be a individual problem).

**Comment by plugg3n** [comment]
@Koenkk The Elko 316 gled dimmers. I have 5, all get included nicely and can be controlled as expected. But i don't get any response back from the dimmer.
*I can controll the dimmer (dim and on/of)
*when i physically push the button I get an response back 
*when i physically dim the wheel, nothing is happening back to z2mqtt.
The status messages in the mqtt says the same brightness payload even if i turn the wheel on the dimmer.
This is not a "big thing" though i would like the status to be correct even if i use them physically.
